src/Dialogs/Task/TaskPointDialog.cpp

```diff
@@ -397,7 +397,7 @@ TaskPointWidget::RefreshView()
 
   {
     StaticString<100> buffer;
-    buffer.Format(_T("%s %s"), name_prefix_buffer.c_str(),
+    buffer.Format(_T("%.80s %s"), name_prefix_buffer.c_str(),
```

Comment:
This way of truncating strings can leave partial UTF-8 sequences in the buffer, which, unlike truncation at the end, cannot be fixed by our text renderer.
Also, it's rather pointless.

Reply:
Dropped it.
Worst (but unlikely) case is that the first %s fills the buffer completely that the second %s is ignored altogether.
So be it.